### ISSUE TITLE
Can't see diff

### ISSUE THREAD

**Issue by mlavandero** [desktop/desktop#1559]
Please summarise the issue in the title, and then use the template below to
fill out the details so we can reproduce the issue on our end.

### Description

When looking at the history in a Repository, and clicking on a specific file in a commit, the diff for that file does not show.  When seeing the Debug tool, there is this error

```
Refused to set unsafe header "User-Agent"
J @ requester.js:40
```

### Version

What version of GitHub Desktop are you running? This is displayed under the
`About GitHub Desktop` menu item. If you are running from source, include
the commit by running `git rev-parse HEAD` from your local repository.

**GitHub Desktop version:** 0.5.3

The operating system you are running on may also help with reproducing the
issue:

 - If you are on macOS, launch `About This Mac` and write down the OS version
   listed.
 - If you are on Windows, open `Command Prompt` and attach the output of this
   command*: `cmd /c ver`

**OS version:** 10.12.5 Beta (16F67a)

### Steps to Reproduce

1. Enter a repository
2. Go to History
3. Select a Commit
4. Select a File

If the issue involves a specific public repository, including the information
about that repository will make it is easier to recreate the issue.

If you think screenshots or a GIF recording will help demonstrate the issue
better, feel free to add them here.

**Expected behaviour:** See the diff of the selected file

**Actual behaviour:** Shows a blank page

**Reproduces how often:** 100%

### Logs

There may be some relevant information in log files generated by GitHub
Desktop:

 - If you are on macOS, attach the most recent log file from:
   `~/Library/Application Support/GitHub Desktop/logs/*.desktop.production.log`
 - If you are on Windows, attach the most recent log file from:
   `%APPDATA%\\GitHub Desktop\\logs\\*.desktop.production.log`

The log files are organized by date, so see if anything was generated for
today's date.

#### Additional Information

Any additional information, configuration or data that might be necessary to
reproduce the issue.

**Comment by joshaber** [comment]
Thanks @mlavandero! I haven't be able to reproduce this locally, which makes me think it has something to do with your specific repository or git configuration.

Could you paste your global git config in here?

**Comment by mlavandero** [comment]
Hi @joshaber, thanks for the response, here goes my `.gitconfig`

```
[push]
        default = current
[merge]
        defaultToUpstream = true
        tool = opendiff
[user]
        name = Mario Lavandero
        email = mlavandero@lemontech.cl
[core]
        excludesfile = /Users/mlavandero/.gitignore_global
[color]
        ui = always
[diff]
        tool = opendiff
[filter "lfs"]
        clean = git-lfs clean %f
        smudge = git-lfs smudge %f
        required = true
```

**Comment by artivilla** [comment]
Having the same issue. Here's my global config file:
```
[alias]
	# Show the diff between the latest commit and the current state
	d = !"git diff-index --quiet HEAD -- || clear; git --no-pager diff --patch-with-stat"
[filter "lfs"]
	clean = git-lfs clean %f
	smudge = git-lfs smudge %f
	required = true
[user]
	name = Arti Villa
	email = artivillachandok@gmail.com
[push]
	default = simple
[color]
	ui = always
[color "diff-highlight"]
	oldNormal = red bold
	oldHighlight = red bold 52
	newNormal = green bold
	newHighlight = green bold 22
[core]
	pager = diff-so-fancy | less --tabs=4 -RFX
[mergetool "Kaleidoscope"]
	cmd = ksdiff --merge --output \"$MERGED\" --base \"$BASE\" -- \"$LOCAL\" --snapshot \"$REMOTE\" --snapshot
	trustexitcode = true
[merge]
	tool = Kaleidoscope
[difftool "Kaleidoscope"]
	cmd = ksdiff --partial-changeset --relative-path \"$MERGED\" -- \"$LOCAL\" \"$REMOTE\"
[difftool]
	prompt = false
[mergetool]
	prompt = false
[diff]
	tool = Kaleidoscope
```
**GitHub Desktop version: 0.5.3**
**OS version: 10.12.4 (16E195)**

**Comment by joshaber** [comment]
This is fixed and available in 0.5.5. Thanks for the reports!
